cdc/sinkv2/metrics/mq/kafka/collector.go

```diff
@@ -91,7 +91,7 @@ func (m *Collector) Run(ctx context.Context) {
 
 func (m *Collector) updateBrokers() {
```

Comment:
```suggestion
func (m *Collector) updateBrokers(ctx context.Context) {
```

There is a context in the caller of `updateBrokers`.